src/shared/components/oncoprint/DataUtils.ts

```diff
@@ -213,11 +219,21 @@ export function fillGeneticTrackDatum(
                 dispMutCounts[oncoprintMutationType] += 1;
                 break;
             case AlterationTypeConstants.STRUCTURAL_VARIANT:
-                structuralVariantCounts += 1;
+                let type: string;
+                if (event.putativeDriver) {
+                    type = 'sv_rec';
```

Comment:
what does `rec` means?

Reply:
this is legacy... back when we used to call these mutations "recurrent" instead of putative driver. it would probably be a good idea to refactor that, I can do it in another PR